java/github/LocalSources.qll

```diff
@@ -0,0 +1,15 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+
+class FileReadAccess extends LocalUserInput {
+  FileReadAccess() {
+    exists(MethodAccess ma |
+      // https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html#getResourceAsStream-java.lang.String-
+      ma.getMethod().getDeclaringType().hasQualifiedName("java.lang", "ClassLoader") and
+      ma.getMethod().hasName("getResourceAsStream") and
```

Comment:
```
ma.getMethod().hasQualifiedName("java.lang", "ClassLoader", "getResourceAsStream")
```
might be more succinct.

Reply:
Is this something that was explicitly requested? There are quite a few other ways to load file xml file contents, that we might want to add as well, possibly in another PR.

Reply:
@zbazztian Will update this to your suggestion.

> There are quite a few other ways to load file xml file contents

Absolutely. This might be better to create a PR in CodeQL itself and a "LocalFileAccess" abstract class

https://github.com/github/codeql/blob/main/java/ql/lib/semmle/code/java/dataflow/FlowSources.qll#L189-L192